zigup.zig

```diff
@@ -292,11 +292,7 @@ pub fn main2() !u8 {
             defer allocator.free(install_dir);
             const compiler_dir = try std.fs.path.join(allocator, &[_][]const u8{ install_dir, version_string });
             defer allocator.free(compiler_dir);
-            if (std.mem.eql(u8, version_string, "master")) {
-                @panic("set default to master not implemented");
-            } else {
-                try setDefaultCompiler(allocator, compiler_dir, .verify_existence);
-            }
+            try setDefaultCompiler(allocator, compiler_dir, .verify_existence);
```

Comment:
I don't think this will work on windows since it doesn't have symlinks (not sure why github actions isn't testing your PR).  I'm fairly certain that what needs to be done is when we see "master", we need to resolve the master version (if master has not been downloaded print a nice error that it hasn't been fetched), and then call setDefaultCompiler with that specific version.

Reply:
Also we should test the case where we run `zigup deafult master` before we have downloaded anything and verify we get a good error message.

If you want me to finish the PR I can but I'll give you a change to implement it if you like.

Reply:
Oh alright. I don't have a windows PC so I couldn't test it and thus I cannot reliably implement it either. So yeah please continue.